configs/terraform/modules/artifact-registry/README.md

```diff
@@ -1,24 +1,61 @@
-artifact_registry_primary_area  = "europe"
+artifact_registry_collection = {
+  modules-internal={
+    name                   = "modules-internal"
+    owner                  = "neighbors"
+    type                   = "production"
+    reader_serviceaccounts = ["klm-controller-manager@sap-ti-dx-kyma-mps-dev.iam.gserviceaccount.com", "klm-controller-manager@sap-ti-dx-kyma-mps-stage.iam.gserviceaccount.com", "klm-controller-manager@sap-ti-dx-kyma-mps-prod.iam.gserviceaccount.com"]
+    writer_serviceaccount  = "kyma-submission-pipeline@kyma-project.iam.gserviceaccount.com"
+  },
+}
 ```
 
-When you use the GCP private image registry, consider the following: 
+If you would like to crete a new Artifact registry please copy an existing registry data then modify the **required** parameters according to your requirements. **!Please, do not delete or update existing registry related data set without knowing what you are doing!**
+
+```terraform
+artifact_registry_collection = {
+    ...
+  <your registry's name>={
+    name                   = "<your registry's name>"
+    owner                  = "<registry owner>"
+    type                   = "<type: development or production>"
+    reader_serviceaccounts = ["<service account 1>", "<service account 2>"]
+  },
+  ...
+}
+```
+
+Additionally you have chance to define optional parameters. Here you can find all parameters you can use:
+
+| Parameter              | Description                                                             | Type         | Required | Default value |
+|------------------------|-------------------------------------------------------------------------|--------------|----------|---------------|
+| name                   | Artifact Registry name                                                  | string       | x        |               |
+| owner                  | Registry Owner Team                                                     | string       | x        |               |
+| type                   | Environment type (development, production)                              | string       | x        |               |
+| reader_serviceaccounts | List of Service Accounts who have `Reader` access on registry           | list(string) | x        |               |
+| writer_serviceaccount  | List of Service Account who has  `Writer`  access on registry           | string       |          | ""            |
+| primary_area           | Primary area (if multi region registry)                                 | string       |          | europe        |
+| multi_region           | Multi region or single region registry                                  | bool         |          | true          |
+| public                 | Is it available for every users from the internet with `Reader` access? | bool         |          | false         |
+| immutable              | Immutable tags?                                                         | bool         |          | false         |
```

Comment:
What does the description mean? Is the meaning: "Are there (any) immutable tags?"

Reply:
This is a special tag. Theoretically it is known by the guys who use it. :-)